src/components/BrandConfigProvider/theme/base/colors.ts

```diff
@@ -1,32 +1,118 @@
-const palette = {
-  neutral200: '#F4F2F1',
-  neutral300: '#D7CEC9',
-  neutral400: '#B6ACA6',
-  neutral600: '#564E4A',
-  neutral800: '#191015',
-
-  blueGray200: '#B0BEC5',
-  blueGray800: '#37474F',
-
-  error100: '#F2D6CD',
-  error500: '#C03403',
-};
+/**
+ * Base (wireFrame-like) color scheme
+ *
+ * The base color scheme uses a popular design system to
+ * dynamically generate all color tokens from the HTML color:
+ * "gray" - #808080	- rgb(128,128,128)
+ */
+const scheme = {
+  light: {
+    primary: 'rgb(0, 104, 116)',
+    onPrimary: 'rgb(255, 255, 255)',
+    primaryContainer: 'rgb(151, 240, 255)',
+    onPrimaryContainer: 'rgb(0, 31, 36)',
+
+    secondary: 'rgb(74, 98, 103)',
+    onSecondary: 'rgb(255, 255, 255)',
+    secondaryContainer: 'rgb(205, 231, 236)',
+    onSecondaryContainer: 'rgb(5, 31, 35)',
+
+    tertiary: 'rgb(82, 94, 125)',
+    onTertiary: 'rgb(255, 255, 255)',
+    tertiaryContainer: 'rgb(218, 226, 255)',
+    onTertiaryContainer: 'rgb(14, 27, 55)',
+
+    error: 'rgb(186, 26, 26)',
+    onError: 'rgb(255, 255, 255)',
+    errorContainer: 'rgb(255, 218, 214)',
+    onErrorContainer: 'rgb(65, 0, 2)',
+
+    background: 'rgb(250, 253, 253)',
+    onBackground: 'rgb(25, 28, 29)',
+
+    surface: 'rgb(250, 253, 253)',
+    onSurface: 'rgb(25, 28, 29)',
+    surfaceVariant: 'rgb(219, 228, 230)',
+    onSurfaceVariant: 'rgb(63, 72, 74)',
+
+    outline: 'rgb(111, 121, 122)',
+    outlineVariant: 'rgb(191, 200, 202)',
```

Comment:
![light](https://user-images.githubusercontent.com/220893/222758805-304946bc-5e5d-41de-ab6c-b6e5a24a3b45.png)

Reply:
We should be able to refer to these colors visually by clicking back to this PR from a vscode context menu near this code.